h2o-core/src/main/java/water/H2O.java

```diff
@@ -2139,6 +2137,16 @@ public static void main( String[] args ) {
     Log.debug("    Start GA: " + (time12 - time11) + "ms");
   }
 
+  /** Find PID of the current process, use -1 if we can find the value */
+  private static void initializePID(){
```

Comment:
Good point. I wanted to do the smallest possible change, however why not improve right away. Thanks! 👍